docs/frontend/tensorflow.rst

```diff
@@ -0,0 +1,226 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+TensorFlow Frontend
+===================
+
+The TensorFlow frontend helps in importing TensorFlow models into TVM.
+
+Supported versions:
+
+- 1.12 and below
+
+Tested models:
+
+- Inception (V1/V2/V3/V4)
```

Comment:
Do we exercise model import from TF with these models? If not, would it be worth adding unit tests to catch importer errors? This does not belong to the scope of this PR, but would be worth doing as follow up.

Reply:
We should definitely do this as a follow-up. A while ago, I actually started working on a BERT unit test, but re-writing it from scratch in TVM looked like it was going to take a long time. I'll pick it back up.

Reply:
I think V1 & V3 are already in the test cases. Adding all may increase CI execution time.